style.css

```diff
@@ -1,3 +1,131 @@
+
+body {
+  background-color: rgb(206, 218, 218);
+  width: 98%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
 #root {
-  color: red;
+  background-color:black;
+  color: rgb(240, 245, 234);
+  width: 95%;
+  margin: 20px 0 20px 0;
+  box-shadow: 0px 0px 10px rgba(49, 54, 58, 0.7);
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
+}
+
+
+#search {
+  width: 95%;
+  border: darkkhaki;
+  background-color: rgba(49, 54, 58, 0.7);
+  border-style: solid;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  color: bisque;
+  margin: 20px;
+  align-content: flex-start;
+  border-radius: 5px;
+}
+
+.home {
+  background-color: rgb(119, 30, 8);
+  color: rgb(247, 243, 237);
+  height: 25px;
+  margin-top: 16px;
+  margin-left: 25px;
+  font-weight: 500;
+  width: 70px;
+}
+
+.selectshow {
+  background-color: burlywood;
+  height: 25px;
+  margin-top: 16px;
+  margin-left: 25px;
+  font-weight: 500;
+  width: 400px;
+}
+
+.selectepisode {
+  background-color: burlywood;
+  height: 25px;
+  margin-top: 16px;
+  margin-left: 25px;
+  font-weight: 500;
+  width: 400px;
+}
+
+input {
+  background-color: burlywood;
+  height: 20px;
+  margin-top: 16px;
+  margin-left: 25px;
+  font-weight: 500;
+  width: 400px;
+}
+
+#container {
```

Comment:
As we learn from CSS course, it is best practice to style the element with  class selector instead of ID selector. I think You may want to redo it  something like that `<div id="container" class="container"></div> `  Having said that, I really like your design and color combination. :)